### PR TITLE
kubecon eu / v2.5.0 post

### PR DESCRIPTION
**Pull request by krancour**
Note that this post won't show up in the Netlify preview because it has Monday's date.

### CODE REVIEW

**Comment by karenhchu** [comment]
Can we link the Brigade User Survey in this blog post? Figured it'd be a good time to push it + during the virtual office hour. Will update shortly with the URL.

**Comment by karenhchu** [comment]
https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR5-_q_vEyFVNofZJNZLyGUBUNjJCNTlMUkVQRUgwNk40ODJJQkRQQTBXQy4u

File: content/2022-05-16-brigade-2.5.0.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+backwards compatible implementation of our Git Initializer component to correct
+our recently discovered incompatibility with Git repositories hosted on
+[Azure DevOps](https://azure.microsoft.com/en-us/services/devops/). And while
+Azure DevOps compatibility is a great improvement in and of itself, the new
+implementation should also be more efficient for certain cases because it never
+fetches more than what it needs from your remote repositories.
+
+And with that, we're charging full steam ahead into KubeCon! Be on the lookout
+for more of our peripheral components (like event gateways) going GA this week,
+and if you're registered for KubeCon, be sure to join us for our virtual office
```
Comment (karenhchu) [inline content/2022-05-16-brigade-2.5.0.md:25]:
```suggestion
and if you're registered for KubeCon, be sure to join us for our [virtual office hour](https://community.cncf.io/e/mjq57f/)
```

File: content/2022-05-16-brigade-2.5.0.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+hours 3:30 - 4:15 PM, Central European Time.
```
Comment (karenhchu) [inline content/2022-05-16-brigade-2.5.0.md:26]:
```suggestion
from 3:30 - 4:15 PM, Central European Time.
```

File: content/2022-05-16-brigade-2.5.0.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (karenhchu) [inline content/2022-05-16-brigade-2.5.0.md:27]:
```suggestion

Additionally, we are looking to learn more about how our community is using Brigade in the real world. Help us out by taking out our [Brigade Use Case Survey](https://forms.office.com/r/jLfGKhekfd) and sharing it with others you know using Brigade. 

```

**Comment by krancour** [comment]
@karenhchu thanks for the feedback. Want to have another look now?

File: content/2022-05-16-brigade-2.5.0.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+[Azure DevOps](https://azure.microsoft.com/en-us/services/devops/). And while
+Azure DevOps compatibility is a great improvement in and of itself, the new
+implementation should also be more efficient for certain cases because it never
+fetches more than what it needs from your remote repositories.
+
+While we have your attention, we're also asking for help in understanding the
+ways _you_ are using Brigade in the real world. Please consider helping us by
+taking our short
+[Brigade Use Case Survey](https://forms.office.com/r/jLfGKhekfd) and sharing it
+with others you know who use Brigade.
```
Comment (karenhchu) [inline content/2022-05-16-brigade-2.5.0.md:27]:
```suggestion
with others you know using Brigade. 
```

**Pull request merged**